src/DataWrangling/JRA55.jl

```diff
@@ -81,23 +110,152 @@ urls = Dict(
 
-    :atmospheric_eastward_velocity => "https://www.dropbox.com/scl/fi/86wetpqla2x97isp8092g/" *
-                                      "RYF.uas.1990_1991.nc?rlkey=rcaf18sh1yz0v9g4hjm1249j0&dl=0",
+    :eastward_velocity => "https://www.dropbox.com/scl/fi/86wetpqla2x97isp8092g/" *
+                          "RYF.uas.1990_1991.nc?rlkey=rcaf18sh1yz0v9g4hjm1249j0&dl=0",
 
-    :atmospheric_northward_velocity => "https://www.dropbox.com/scl/fi/d38sflo9ddljstd5jwgml/" *
-                                       "RYF.vas.1990_1991.nc?rlkey=f9y3e57kx8xrb40gbstarf0x6&dl=0",
+    :northward_velocity => "https://www.dropbox.com/scl/fi/d38sflo9ddljstd5jwgml/" *
+                           "RYF.vas.1990_1991.nc?rlkey=f9y3e57kx8xrb40gbstarf0x6&dl=0",
 )
 
+compute_bounding_nodes(::Nothing, ::Nothing, LH, hnodes) = nothing
+compute_bounding_nodes(bounds, ::Nothing, LH, hnodes) = bounds
+
+function compute_bounding_nodes(::Nothing, grid, LH, hnodes)
+    hg = hnodes(grid, LH())
+    h₁, h₂ = extrema(hg)
+    return h₁, h₂
+end
+
+function compute_bounding_indices(::Nothing, hc)
+    Nh = length(hc)
+    return 1, Nh
+end
+
+function compute_bounding_indices(bounds, hc)
+    h₁, h₂ = bounds
+    Nh = length(hc)
+
+    # The following should work. If ᵒ are the extrema of nodes we want to
+    # interpolate to, and the following is a sketch of the JRA55 native grid,
+    #
+    #      1         2         3         4         5
+    # |         |         |         |         |         |
+    # |    x  ᵒ |    x    |    x    |    x  ᵒ |    x    |
+    # |         |         |         |         |         |
+    # 1         2         3         4         5         6 
+    #
+    # then for example, we should find that (iᵢ, i₂) = (1, 5).
+    # So we want to reduce the first index by one, and limit them
+    # both by the available data. There could be some mismatch due
+    # to the use of different coordinate systems (ie whether λ ∈ (0, 360)
+    # which we may also need to handle separately.
+    i₁ = searchsortedfirst(hc, h₁)
+    i₂ = searchsortedfirst(hc, h₂)
+    i₁ = max(1, i₁ - 1)
+    i₂ = min(Nh, i₂)
+
+    return i₁, i₂
+end
+
+infer_longitudinal_topology(::Nothing) = Periodic
+
+function infer_longitudinal_topology(λbounds)
+    λ₁, λ₂ = λbounds
+    TX = λ₂ - λ₁ ≈ 360 ? Periodic : Bounded
+    return TX
+end
+
+
+function compute_bounding_indices(longitude, latitude, grid, LX, LY, λc, φc)
+    λbounds = compute_bounding_nodes(longitude, grid, LX, λnodes)
+    φbounds = compute_bounding_nodes(latitude, grid, LY, φnodes)
+
+    i₁, i₂ = compute_bounding_indices(λbounds, λc)
+    j₁, j₂ = compute_bounding_indices(φbounds, φc)
+    TX = infer_longitudinal_topology(λbounds)
+
+    return i₁, i₂, j₁, j₂, TX
+end
+
+# Convert dates to range until Oceananigans supports dates natively
+function jra55_times(native_times, start_time=DateTimeNoLeap(1900, 01, 01))
+    Nt = length(native_times)
+    Δt = native_times[2] - native_times[1] # assume all times are equispaced
```

Comment:
what if `native_times = 1:1`?

Reply:
`test_downloading.jl` errors because of this (and possibly this implicit assumption elsewhere?)

Reply:
In Oceananigans we assume that a `FieldTimeSeries` with values in memory has at least 2 data points

https://github.com/CliMA/Oceananigans.jl/blob/f7042b738da746cbb6ba752e01c3894b09b6f1a4/src/OutputReaders/field_time_series.jl#L45-L48

I think it is a valid assumption given that a field time series with one point is just a field. 
On the other hand, we have to options
(1) fix the tests in `ClimaOcean.jl` keeping the assumption that `Ntimes > 2` (easy option) 
(2) relax the assumption in `Oceananigans.jl` (harder to maintain but more general?)

I vote for (1)